prow/plugins/plugins.go

```diff
@@ -291,6 +291,25 @@ func (pa *ConfigAgent) Load(path string, supplementalPluginConfigDirs []string,
 				return nil
 			}
 
+			relPath, err := filepath.Rel(supplementalPluginConfigDir, path)
```

Comment:
This won't work in a configmap

Reply:
The file will be a symbolic link to `..data/...`. We will still be able to read the path without search for the original link since we are not resolving it. Also, the above code ignores `..` prefixes for this reason.